flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -730,15 +748,49 @@ public CompletableFuture<CoordinationResponse> deliverCoordinationRequestToCoord
                                                         + " does not exist")));
     }
 
+    @Override
+    public JobResourceRequirements requestJobResourceRequirements() {
+        final JobResourceRequirements.Builder builder = JobResourceRequirements.newBuilder();
+        for (JobInformation.VertexInformation vertex : jobInformation.getVertices()) {
+            builder.setParallelismForJobVertex(vertex.getJobVertexID(), 1, vertex.getParallelism());
```

Comment:
Here, the lower bound is always `1`, and `SlotAllocator` also ignores it.

Should we throw `UnsupportedOperationException` if the requested lower bound is higher?

Reply:
👍 This is indeed something that needs to be addressed. I'll change (so AS respects the setting) that in a separate PR to avoid inflating this one any further.

I've created https://issues.apache.org/jira/browse/FLINK-31476 to track it.

I think it's OK to ignore the setting until then, WDYT?

Reply:
Sounds good, thanks!